.github/workflows/sanity-test.yaml

```diff
@@ -355,14 +355,79 @@ jobs:
         with:
           service: sharepoint
           kind: incremental
-          backup-args: '--site "${{ vars.CORSO_M365_TEST_SITE_URL }}"'
+          backup-args: '--site "${{ vars.CORSO_M365_TEST_SITE_URL }}" --data library'
           restore-args: '--folder ${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-sharepoint.outputs.result }}'
           restore-container: '${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-sharepoint.outputs.result }}'
           log-dir: ${{ env.CORSO_LOG_DIR }}
           with-export: true
 
 ##########################################################################################################################################
 
+# Sharepoint Lists
+
+      # generate new entries for test
+      - name: SharePoint - Create new data
+        id: new-data-creation-sharepoint
+        timeout-minutes: 30
+        working-directory: ./src/cmd/factory
+        run: |
+          suffix=$(date +"%Y-%m-%d_%H-%M-%S")
+
+          go run . sharepoint lists  \
+            --site ${{ vars.CORSO_M365_TEST_SITE_URL }} \
+            --user ${{ env.TEST_USER }} \
+            --secondaryuser  ${{ env.CORSO_SECONDARY_M365_TEST_USER_ID }} \
+            --tenant ${{ secrets.TENANT_ID }} \
+            --destination ${{ env.RESTORE_DEST_PFX }}$suffix \
+            --count 4 |
+            awk 'NR > 1 {print $2}' | tr '\n' ',' | sed -e 's/,$//' -e 's/^/result=/' |
```

Comment:
Converts CLI o/p,

From:
```
Restoring to folder Corso_Test_Sanity_dttm
Generating 4 lists items in Corso_Test_Sanity_dttm

  ID            List                                                         Items  Created               Modified            
  2a1e0850638b  Corso_Test_Sanity_dttm_5dfbe852-ad79-4edc-bce2-5d2b4fc65e97  20     2024-01-15T12:50:21Z  2024-01-15T12:50:21Z
  8e49fd6a0cd6  Corso_Test_Sanity_dttm_2028ba03-8dc3-45cc-b989-f0ba3a38bd94  20     2024-01-15T12:50:29Z  2024-01-15T12:50:29Z
  fdc2f3046708  Corso_Test_Sanity_dttm_1b4be031-53a2-425c-b988-1e401e3ef712  20     2024-01-15T12:50:38Z  2024-01-15T12:50:38Z
  db744ba333e7  Corso_Test_Sanity_dttm_63464c42-7c6a-49b6-9cea-cc350d1a6912  20     2024-01-15T12:50:45Z  2024-01-15T12:50:46Z
```

To:
`Corso_Test_Sanity_dttm_543beed0-aa14-4780-b5b7-cc87fba12f38,Corso_Test_Sanity_dttm_697c0cc1-60da-440c-9e96-65c7ba430098,Corso_Test_Sanity_dttm_297d0b08-13c4-4f13-82cb-000fcb33f47b,Corso_Test_Sanity_dttm_f02f0397-c371-4584-97b2-afe8fa8d6e14`